Python.wiki/Home.md

```diff
@@ -0,0 +1,6 @@
+## Python Algorithms
```

Comment:
Guess you're trying to fix multiple issues in a single PR ?

Reply:
Yes, I suppose that is true. In fixing the wiki issue, the doctests issue was highlighted. Both are addressed here